javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/Difference.java

```diff
@@ -510,7 +510,7 @@ private void applyKeptDiffElement(Kept kept, TextElement originalElement, boolea
                     originalIndex++;
                     diffIndex++;
                 } else {
-                    throw new UnsupportedOperationException("kept " + kept.getElement() + " vs " + originalElement);
+                    originalIndex++;
```

Comment:
Can you be more specific here before jp threw an exception but with your solution it always keep the token?